packages/server/src/api/routes/public/index.ts

```diff
@@ -120,6 +121,7 @@ function applyRoutes(
 }
 
 applyRoutes(appEndpoints, PermissionType.APP, "appId")
+applyRoutes(metricEndpoints, PermissionType.APP, "appId")
```

Comment:
Do we need the `PermissionType.APP, "appId"` here?  There's no `appId` in the `"/metrics"` path. 
We should lock the new endpoint down as an builder or admin only operation